jira/client.py

```diff
@@ -2742,9 +2741,11 @@ def search_users(self, user, startAt=0, maxResults=50, includeActive=True, inclu
         :rtype: ResultList
         """
         params = {
-            'username': user,
+            'query': query,
             'includeActive': includeActive,
             'includeInactive': includeInactive}
+        if query:
```

Comment:
The rationale here is that empty `query` implies that `accountId` is used for the query.